### ISSUE TITLE
About the backbone model

### ISSUE THREAD

**Issue by BearVic** [facebookresearch/maskrcnn-benchmark#214]
## ❓ Questions and Help
Do you implement the model using VGG16 as the backbone?

**Comment by fmassa** [comment]
No, VGG16 backbone is not implemented, and it is not in the plans, but contributions are welcome! :-)

**Comment by BearVic** [comment]
@fmassa I'd glad to do that! It is a great opportunity for me to practice! However, I encountered one issue when installing the environment:
```
nvcc fatal: 'std' not found
Error, status 255
```
I read the answers of the similar question in the Issues and tried:
gcc 5.2 6.2
cuda 9.0 9.2
But they did not work.
Could you help me with that?

**Comment by fmassa** [comment]
This is probably a wrong version of CUDA / nvcc being picked up, see https://github.com/NervanaSystems/neon/issues/150 for example

**Comment by fmassa** [comment]
you could try uninstalling the version that you have in `/usr/local/cuda`, if you have sudo access.
Those conflicting libraries are going to be annoying when compiling PyTorch from source as well I suppose.

**Comment by fmassa** [comment]
can you try specifying `CUDA_HOME` env var to point to your CUDA install?

```
CUDA_HOME=software/cuda-toolkit/9.0.176/ python setup.py build develop
```

**Comment by BearVic** [comment]
@fmassa It works!!! Thank you so much!! Now I can learn this codes and try VGG. I am so excited! Thanks again!
